kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -74,120 +85,98 @@ def main() -> None:
     parser = _create_argument_parser()
     args = parser.parse_args()
     logging.basicConfig(level=_loglevel(args), format=_LOG_FORMAT)
+    options = generate_command_options({key: val for (key, val) in vars(args).items() if val is not None})
 
     executor_name = 'exec_' + args.command.lower().replace('-', '_')
     if executor_name not in globals():
         raise AssertionError(f'Unimplemented command: {args.command}')
 
     execute = globals()[executor_name]
-    execute(**vars(args))
+    execute(options)
```

Comment:
It would be cool if we instead had something like:

```
kevm_cli = KEVMCLI(args)
```

And it just kinda internalizes all the logic. Then all teh `exec_...` methods can be moved there, and they can be registered explicitely for each subcommand inside the class.